saleor/core/db_routers.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+
+class PrimaryReplicaRouter:
+    def db_for_read(self, model, **hints):
+        """Read from replica."""
+        return settings.DATABASE_CONNECTION_REPLICA_NAME
```

Comment:
Won't this switch our whole read traffic to replicas? 
In my opinion, we shouldn't switch all read traffic. If we want to read from a replica we should do it explicitly. 
When we switch to always read from replica we could cause that situation when we respond to mutation from replica before this database receives propagated data from primary database.

Reply:
Removed.